clash-cores/test/Test/Cores/Crc.hs

```diff
@@ -61,7 +61,9 @@ proxyToSNat :: KnownNat n => Proxy n -> SNat n
 proxyToSNat _ = SNat
 
 nearestCleanDivisor :: Integer -> Integer -> Integer
-nearestCleanDivisor a b =  List.head $ List.sortOn (\i -> abs (i - b)) candidates
+nearestCleanDivisor a b = case List.sortOn (\i -> abs (i - b)) candidates of
```

Comment:
Why was this changed? The list should never be empty. Simply ro silence a newly introduced `List.head` usage warning?

Reply:
Yeah, head and tail give warnings as of base-4.19